core/domain/feedback_jobs_one_off.py

```diff
@@ -51,6 +51,8 @@ def map(thread_model):
         if cache_updated:
             thread_model.put(update_last_updated_time=False)
             yield ('Updated', 1)
+        else:
+            yield ('Correct', 1)
```

Comment:
Perhaps "Already up-to-date" might be a better key.

Reply:
Done.